frontend/apps/desktop/src/find-in-page.tsx

```diff
@@ -1,22 +1,26 @@
 import {StyleProvider} from '@/app-context-provider'
-import {useStream} from '@shm/ui/use-stream'
 import '@tamagui/core/reset.css'
 import '@tamagui/font-inter/css/400.css'
 import '@tamagui/font-inter/css/700.css'
-import React from 'react'
+import React, {useEffect, useState} from 'react'
 import ReactDOM from 'react-dom/client'
 import {FindInPage} from './pages/find-in-page'
 
 function FindInPageView() {
-  // @ts-expect-error
-  const darkMode = useStream<boolean>(window.darkMode)
+  const [darkMode, setDarkMode] = useState(false)
```

Comment:
it looks like darkMode will always start off as false? This is the reason we have useSyncExternalStore inside useStream

And, what is wrong with useStream?

Reply:
is this important for this PR to be merged?

Reply:
No I'm happy to merge anyways.

But I'm trying to understand why you moved to useState+useEffect because this is not the right approach in my opinion. useSyncExternalStore is meant for this, and that is how useStream works. And you deleted useStream here. Why?